### PR TITLE
Adding draft "Material Core" for testing.

### PR DESCRIPTION
**Pull request by tucotuco**

### CODE REVIEW

**Comment by tucotuco** [comment]
Addresses https://github.com/gbif/rs.gbif.org/issues/105

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title='Darwin Core Occurrence'
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:7]: This should be Darwin Core Material Entity?

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    name='Occurrence' namespace='http://rs.tdwg.org/dwc/terms/'
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:8]: MaterialEntity?

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    rowType='http://rs.tdwg.org/dwc/terms/Occurrence'
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:9]: MaterialEntity?

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    dc:issued='2022-02-02'
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:10]: 2023-04-29?

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    dc:subject='dwc:Event dwc:Taxon'
+    dc:relation='http://rs.tdwg.org/dwc/terms/Occurrence'
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:12]: MaterialEntity

File: sandbox/core/dwc_material_2023-04-29.xml [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+    dc:description='The category of information pertaining to the existence of an Organism (sensu http://rs.tdwg.org/dwc/terms/Organism) at a particular place at a particular time.'>
```
Comment (timrobertson100) [inline sandbox/core/dwc_material_2023-04-29.xml:13]: Represents a material entity... etc

**Comment by timrobertson100** [comment]
I've added some single-line comments but also suggest to delete the MaterialSample completely

**Comment by tucotuco** [comment]
I believe your concerns have bee met with https://github.com/gbif/rs.gbif.org/pull/107/commits/8ef7ad2e6e3324b8f5bd4441a284385e33adfb17. Please check.

**Review by timrobertson100 — APPROVED** [review]

**Pull request merged**